lib/puppet/indirector/json.rb

```diff
@@ -52,7 +52,7 @@ def load_json_from_file(file, key)
     json = nil
 
     begin
-      json = File.read(file)
+      json = File.read(file).force_encoding(Encoding::UTF_8)
```

Comment:
@joshcooper Yeah, I can make that change. I've been banging on that facter ticket this week so far.

The YAML terminus probably has the same problem, I'll look in to it as well